### DISCUSSION TITLE
Does BPL only support 32-bit applications?

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#25] [Q&A] [answered]
After I had finished the entity class TDocument, I wanted to do a "quick-and-dirty" test and integrate it quickly. That's what I did after information 4.5.3 and 4.5.4. I got the following error message. (see screenshot) I had compiled everything as 64-bit at the time. But including the BPL tells me that only 32-bit applications are supported. What am I doing wrong at this point? I would like to create a complete 64-bit application. So far, it works so far if everything is created as a 32-bit application.
<img width="595" alt="image" src="https://github.com/holgerflick/flixaccounting/assets/681882/079d04bd-3e67-40e8-b695-414d17f7fa50">
It can be run as a 32-bit application and so far everything works. Now I can continue writing the Entity model.
<img width="349" alt="image" src="https://github.com/holgerflick/flixaccounting/assets/681882/91d54c4e-29af-4e0a-946b-11c8507df722">

**Comment by holgerflick** [comment, marked as answer]
Thomas, the Delphi IDE is a 32-Bit application. Thus, if you want to load a package into the IDE it has to be 32 Bit. It has nothing to do with the resulting application which can be 64 Bit. However, we link the units directly. The package is only for design-time and should stay 32 Bit as long as the Delphi IDE is.
